pages/event-resources.jsx

```diff
@@ -193,8 +193,8 @@ var EventsResources = React.createClass({
           alt="Brand Palette Image"
           src1x="/img/event-resources-page/resource-thumbnails-03.png"
           src2x="/img/event-resources-page/resource-thumbnails-03@2x.png">
-            <LogoAssetLink href="">Download EPS</LogoAssetLink>
-            <LogoAssetLink href="">Download PNG</LogoAssetLink>
+            <LogoAssetLink href="https://party.webmaker.org/party-resources/MakerPartyBrandPalette.eps">Download EPS</LogoAssetLink>
+            <LogoAssetLink href="https://party.webmaker.org/party-resources/MakerPartyBrandPalette.png">Download PNG</LogoAssetLink>
```

Comment:
points to party.webmaker.org for now (see https://github.com/mozilla/teach.webmaker.org/issues/828#issuecomment-97208662)